tests/tests/test_mtls.py

```diff
@@ -237,7 +208,7 @@ def common_test_mtls_enterprise(self, env, algorithm=None, use_hsm=False):
         # start the Mender client
         logger.info("starting the client.")
         env.device.run("systemctl daemon-reload")
-        env.device.run("systemctl start %s" % client_service_name)
+        env.device.run("systemctl start mender-authd")
```

Comment:
same as comment below, nit, but maybe it would be nice to have something like `env.device.get_client_service_name` but for mender-authd?